hw/ip/ascon/pre_dv/ascon_tb/ascon_sim.core

```diff
@@ -1,5 +1,5 @@
 CAPI=2:
-# Copyright lowRISC contributors.
+# Copyright lowRISC contributors (OpenTitan project).
```

Comment:
Can you please merge this commit into the previous ones, i.e., where you add the corresponding files?